examples/http-server/main.go

```diff
@@ -7,16 +7,21 @@ import (
 
 	"github.com/corazawaf/coraza/v3"
 	txhttp "github.com/corazawaf/coraza/v3/http"
+	ctypes "github.com/corazawaf/coraza/v3/types"
 )
 
 func hello(w http.ResponseWriter, req *http.Request) {
-	fmt.Fprintf(w, "hello world, not disrupted.\n")
+	// The server generates the response
+	w.Header().Set("Status", "200")
+	w.Header().Set("Protocol", "HTTP/1.1")
```

Comment:
```suggestion
```

There's no Protocol header in HTTP. This example is always HTTP/1 since we don't initialize the `http2` package with the handle function

Reply:
It has been an improper attempt to pass these values to `ProcessResponseBody`